tests/bgp_commands_test.py

```diff
@@ -97,6 +97,10 @@
 Error: bgp summary from bgp container not in json format
 """
 
+show_error_no_neighbor = """\
+% No BGP neighbors found in VRF default
```

Comment:
For IPV6 case This needs to be changed to:
**No IPv6 neighbor is configured**
For IPV4 case this needs to be changed to:
**No IPv4 neighbor is configured**

Reply:
updated in latest commit